src/update-script.rb

```diff
@@ -63,7 +63,7 @@
       "url" => url,
       "token" => ":#{ENV["SYSTEM_ACCESSTOKEN"]}", # do not forget the colon
     }
-  elsif package_manager == "maven"
+  elsif package_manager == "gradle"
```

Comment:
Shouldn't we check for both `maven` and `gradle` so as to add the private maven feed?